src/seedu/addressbook/commands/Command.java

```diff
@@ -60,6 +60,8 @@ protected ReadOnlyPerson getTargetPerson() throws IndexOutOfBoundsException {
         return relevantPersons.get(getTargetIndex() - DISPLAYED_INDEX_OFFSET);
     }
 
+
+
```

Comment:
Stray space? It is a good practice to check the diff of the PR and fix any problems before submitting it.